fe/fe-core/src/main/java/org/apache/doris/nereids/rules/exploration/mv/PredicatesSplitter.java

```diff
@@ -102,9 +104,9 @@ public Void visit(Expression expr, Void context) {
 
     public Predicates.SplitPredicate getSplitPredicate() {
         return Predicates.SplitPredicate.of(
-                equalPredicates.isEmpty() ? null : ExpressionUtils.and(equalPredicates),
-                rangePredicates.isEmpty() ? null : ExpressionUtils.and(rangePredicates),
-                residualPredicates.isEmpty() ? null : ExpressionUtils.and(residualPredicates));
+                equalPredicates.isEmpty() ? null : ExpressionUtils.and(Lists.newArrayList(equalPredicates)),
```

Comment:
what's the opt point for these kind of change?

Reply:
change ExpressionUtil.and/or implementation,  also change its argument type from Collation<Expression> to List<Expression>,  because new And/Or require List<Expression>.